raiden-dapp/.eslintrc.js

```diff
@@ -3,11 +3,9 @@ const path = require('path');
 module.exports = {
   root: true,
   env: {
-    node: true,
+    node: true
```

Comment:
Hmm, looking at this, it doesn't make sense. We don't run the dApp on a node environment, right? we target browsers instead.

Reply:
Well, this was just an `eslint --fix`, the value was already there, but tsc/babel uses some types/logic from Node, maybe that's the reason. [Here's the docs](https://eslint.org/docs/user-guide/configuring#specifying-environments) about these env, feel free to reevaluate.